sympy/core/numbers.py

```diff
@@ -469,16 +467,26 @@ class Float(Number):
     >>> Float(3)
     3.00000000000000
 
-    Floats can be created from a string representations of Python floats
-    to force ints to Float or to enter high-precision (> 15 significant
-    digits) values:
+    Creating Floats from strings (and Python ``int`` and ``long`` types)
+    typically uses approximately 15 significant digits of precision; however
```

Comment:
Perhaps delete the questionable section we discussed and rewrite this section as:

```
Creating Floats from strings (and Python ``int`` and ``long`` types)
will give a minimum precision of 15 digits, but the precision will
automatically increase to capture all digits entered.

>>> Float(1)
1.00000000000000
>>> 10**20
100000000000000000000L
>>> Float(_)
100000000000000000000.

Large *floating-point* numbers are truncated to 15 digits of precision,
however:

>>> Float(1e20)
1.00000000000000e+20

The desired number of digits can also be specified:
```